src/__tests__/JWE.test.ts

```diff
@@ -1,58 +1,87 @@
 import { decryptJWE, createJWE, Encrypter } from '../JWE'
 import vectors from './jwe-vectors.js'
-import { xc20pDirEncrypter, xc20pDirDecrypter, x25519Encrypter, x25519Decrypter } from '../xc20pEncryption'
-import { decodeBase64url } from '../util'
+import { xc20pDirEncrypter, xc20pDirDecrypter, x25519Encrypter, x25519Decrypter,
+         x25519AuthEncrypter, x25519AuthDecrypter } from '../xc20pEncryption'
+import { bytesToBase64, decodeBase64url, encodeBase64url } from '../util'
 import * as u8a from 'uint8arrays'
 import { randomBytes } from '@stablelib/random'
 import { generateKeyPairFromSeed } from '@stablelib/x25519'
 
 describe('JWE', () => {
   describe('decryptJWE', () => {
-    describe('Direct encryption', () => {
-      test.each(vectors.dir.pass)('decrypts valid jwe', async ({ key, cleartext, jwe }) => {
+    describe('ECDH-ES (X25519), Direct Mode with XC20P content encryption', () => {
```

Comment:
Change back.

Reply:
yep, same as above. Will change it to `Direct Encryption with XC20P`